scripts/india_edm/IndiaEnergy_Coal/test_data/test.mcf

```diff
@@ -0,0 +1,180 @@
+Node: dcid:Annual_Consumption_Coal_ElectricityGeneration_CoalProducts
+typeOf: dcs:StatisticalVariable
+populationType: dcs:Coal
+measuredProperty: dcs:consumption
+measurementQualifier: dcs:Annual
+statType: dcs:measuredValue
+energySource: dcs:CoalProducts
```

Comment:
Yes, these properties work for this SV. But there are other SVs like [Annual_Consumption_Coal_ConstructionIndustry_CoalProducts](https://github.com/datacommonsorg/data/blob/ea13e41d8fbac17e454b561d8bc8e4c726ca4368/scripts/india_edm/IndiaEnergy_Coal/test_data/test.mcf#L37) that find its application in Construction Industry (cement production in particular). Should we change the populationType for each SV then? In that case, populationType should be related to the consumingSector - like it will be `populationType: Product` for this SV.

If I may suggest, can we do `populationType: Energy` for all fuel sources in this dataset? Because all types of coal consumption result in generating Energy. I think that will be a more appropriate value.

Reply:
Yes, that makes sense. Where we can use Electricity, or a more specific type, we should use that (like in this example). For consumption, keeping it as `populationType: Coal` seems appropriate.